README.md

```diff
@@ -143,7 +143,7 @@ The launcher is summoned with `Super` + `/`. The search list displays matching w
 By default, the launcher searches windows and applications. However, you can designate a special launch mode using one of the supported prefixes:
 
 - `:`: Execute a command in `sh`
-- `t:`: Execute a command in `sh` in a terminal
```

Comment:
`t` does need to be followed by a `:` for this to work so I think it is correct as it is currently listed.